### DISCUSSION TITLE
Watchlist Count

### DISCUSSION THREAD

**Discussion by dkrvn** [trakt/showly#830] [Ideas / Feature Requests]
Hi, I noticed that for the tabs "My Shows" and "My Movies", there are numbers to show the total number of shows and movies respectively. I was wondering if it is possible to indicate the total number of shows/movies for the watchlist tabs too? Thanks!

**Comment by michaldrabik** [comment]
Yes it would make sense of course. Hoping to add it in future. Just need to think of how to nicely design it into watchlist tabs.

**Reply by dkrvn** [reply]
Hey, just wanted to thank you for implementing this! :)

I'll close the discussion now.

**Comment by blackholeearth** [comment]
İ made less verbose, collection ui .
This could be applied to  collection > " my_shows/watchlist/ hidden " 

1) Move sort and item count , below.
2) removed borders from sort. 
3) removed filter title . (which is duplicate.)

![showly collection ui improv](https://github.com/michaldrabik/showly-2.0/assets/7512105/8417733b-d37a-4fba-b02b-bf851d9beb21)
